r/src/config.cpp

```diff
@@ -34,4 +35,16 @@ std::vector<std::string> runtime_info() {
   return {info.simd_level, info.detected_simd_level};
 }
 
+// [[arrow::export]]
+void set_timezone_database(cpp11::strings path) {
+  auto paths = cpp11::as_cpp<std::vector<std::string>>(path);
+  if (path.size() != 1) {
+    cpp11::stop("Must provide a single path to the timezone database.");
+  }
+
+  arrow::ArrowGlobalOptions options;
```

Comment:
```suggestion
  arrow::GlobalOptions options;
```

Just looking at the CI failures, and I think this is a typo?

Reply:
Yes, forgot to update the R portion after getting the C++ tests passing. Thanks!